File: Modul-2/Python/PRAK204-2310817110006-Adrian_Bintang_Saputera.py
print("Input")
angka = input().split()
pi = 3.142857

if len(angka) == 1:
    jari_jari = float(angka[0])
    tinggi = float(input())
elif len(angka) == 2:
    jari_jari, tinggi = map(float, angka)

volume = jari_jari**2 * pi * tinggi
luas_permukaan = 2 * ((pi * jari_jari) * (jari_jari + tinggi))
keliling = 2 * pi * jari_jari

print("\nOutput")
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas_permukaan:.2f}")
print(f"Keliling = {keliling:.2f}")

File: Modul-2/C/PRAK204-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>
#include <math.h>

#define PI 3.142857

int main()
{
    double jariJari, tinggi;

    printf("Input\n");
    scanf("%lf %lf", &jariJari, &tinggi);

    double volume = PI * pow(jariJari, 2) * tinggi;
    double luasPermukaan = 2 * PI * jariJari * (jariJari + tinggi);
    double keliling = 2 * PI * jariJari;

    printf("\nOutput\n");
    printf("Volume = %.2lf\n", volume);
    printf("Luas = %.2lf\n", luasPermukaan);
    printf("Keliling = %.2lf\n", keliling);

    return 0;
}
